QEfficient/generation/text_generation_inference.py

```diff
@@ -526,39 +544,44 @@ def run_prefill(self, prompt, generation_len, prefill_logit_bs=1, decode_batch_i
             position_ids (array): The position IDs.
             generation_len (int): The generation length.
         """
+        # Skip inputs/outputs
```

Comment:
The skip buffer is used to clear past input and output values retained in the device. This process should be performed only once when loading a model and should be included in the setup method or the __init__ method within TextGeneration.

Reply:
Moved it to init method of QEffTextGenerationBase where session is created (qpc loaded).